config/locales/en/placements/schools/mentors.yml

```diff
@@ -0,0 +1,10 @@
+en:
+  placements:
+    schools:
+      mentors:
+        show:
+          attributes:
+            mentors:
+              first_name: First name
+              last_name: Last name
```

Comment:
So I think you can delete these two lines.

Reply:
resolved in [3852d92](https://github.com/DFE-Digital/itt-mentor-services/pull/274/commits/3852d927370d77ecb3dd252d00e4ec3e140b9852)